src/quo/components/bottom_sheet/view.cljs

```diff
@@ -1,216 +0,0 @@
-(ns quo.components.bottom-sheet.view
```

Comment:
I guess it is ok to delete the old implementation, as the `config/new-ui-enabled?` false is also not working.
(maybe we can get rid of the new-ui-enabled? flag). @flexsurfer

Reply:
ok, then i'll add the old bottom-sheet component back into quo namespace

Reply:
please revert this

Reply:
Unfortunately, i'm not at my macbook right now and VS Code Online terminal is not working either. Could you possibly revert these changes? @Parveshdhull